live/js.go

```diff
@@ -0,0 +1,198 @@
+package live
+
+import (
+	"encoding/json"
+	"strings"
+)
+
+// DefaultTransitionDuration_ms is the default duration, in milliseconds, of live.JS transitions.
+const DefaultTransitionDuration_ms = int32(200)
+
+type op struct {
+	kind string
+	args map[string]any
+}
+
+// JS provides a way to precompose simple client-side DOM changes that don't require a round-trip to the server.
+// Create a JS struct, call its methods to build up a command, and then render it as the value of a phx-* attribute.
+type JS struct {
+	ops []*op
+}
+
+// Hide hides elements.
+// When the action is triggered on the client, phx:hide-start is dispatched to the hidden elements. After the time specified by :time, phx:hide-end is dispatched.
```

Comment:
s/:time/opts.Time/

Reply:
This is what I get for copying straight from the hex docs!